databricks/koalas/__init__.py

```diff
@@ -106,6 +106,13 @@ def _auto_patch():
     import os
     import logging
 
+    from pyspark.sql import dataframe as df, functions as F
+    from databricks.koalas import functions
+
+    for name in functions.__all__:
+        if not hasattr(F, name):
+            setattr(F, name, getattr(functions, name))
```

Comment:
Hm.. should we auto-patch? I am less sure on this yet as `percentile_approx` seems a Spark function and auto-patching it in Koalas seems a bit counterintuitive.